src/helper/UserSession.cpp

```diff
@@ -110,6 +111,22 @@ namespace SDDM {
             }
         }
 
+        // enter Linux namespaces
+        for (const QString &ns: mainConfig.Namespaces.get()) {
+            qInfo() << "Entering namespace" << ns;
+            int fd = ::open(qPrintable(ns), O_RDONLY);
+            if (fd < 0) {
+                qCritical("open(%s) failed: %s", qPrintable(ns), strerror(errno));
+                exit(Auth::HELPER_OTHER_ERROR);
+            }
+            if (setns(fd, 0) != 0) {
```

Comment:
not an expert of namespaces but shouldn't the second argument be CLONE_NEWNET?

Reply:
Indeed, I documented that `Namespaces=` joins network namespaces, but wrote the code that allows to join any kind of namespace. I do not see any benefits from this limitation, so I'd rather update the documentation.